setup.py

```diff
@@ -7,7 +7,7 @@
 
 setup(
     name = 'OpenFisca-France',
-    version = '19.0.0',
+    version = '19.0.0rc0,
```

Comment:
@Anna-Livia je crois qu'il manque un `'`.

Reply:
There is [no point](https://www.python.org/dev/peps/pep-0440/#pre-releases) in Python style versionning 🙂

Reply:
https://packaging.python.org/tutorials/distributing-packages/#standards-compliance-for-interoperability

Reply:
However this should be reverted before merge.